lib/portlayer/storage/compute/archive.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// StatPath will use Guest Tools to stat a given path in the container
+
+package compute
+
+import (
+	"context"
+	"io"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/vmware/govmomi/guest"
+	"github.com/vmware/govmomi/vim25/soap"
+	"github.com/vmware/govmomi/vim25/types"
+	"github.com/vmware/vic/lib/portlayer/exec"
+	"github.com/vmware/vic/pkg/trace"
+)
+
+func FileTransferFromGuest(ctx context.Context, vc *exec.Container, path string) (io.Reader, error) {
+	defer trace.End(trace.Begin(vc.Config.Name))
+
+	// set up file manager
+	client := vc.VIM25Reference()
+	filemgr, err := guest.NewOperationsManager(client, vc.VMReference()).FileManager(ctx)
+	if err != nil {
+		log.Errorf("OPS MGR ERR: %#v", err)
```

Comment:
`%s` instead of `%#v` err is usually interpreted either by it's type or by err.Error